_studio/mfx_lib/shared/src/mfx_mpeg2_encode_vaapi.cpp

```diff
@@ -176,7 +159,6 @@ namespace
         } else
           assert(!"Unknown FrameRateCode appeared.");
 
-        //sps.aspect_ratio_information = CalculateAspectRatio(sps.picture_width, sps.picture_height);
```

Comment:
The call of `CalculateAspectRatio` is replaced by the `AspectRatio`  a member of a `ENCODE_SET_SEQUENCE_PARAMETERS_MPEG2` structure line below